pyro/distributions/log_normal_negative_binomial.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+from numpy.polynomial.hermite import hermgauss
+
+import torch
+from torch.distributions import constraints
+
+from pyro.distributions import NegativeBinomial, TorchDistribution
```

Comment:
import these from their specific modules:
```py
from pyro.distributions.torch_distribution import TorchDistribution
from pyro.distributions.torch import NegativeBinomial
```
That should resolve the circular import issues. The problem was that `pyro.distributions` imports this module and vice versa. Does that make sense?

Reply:
yup thanks!